docs/concepts/html/elements.md

```diff
@@ -89,6 +89,21 @@ html! {
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+## Optional attributes for HTML elements
+
+Most HTML attributes can be marked as optional by placing a `?` in front of
+the `=` sign. This makes them accept the same type of value as otherwise, but
+wrapped in an `Option<T>`:
+```rust
+let maybe_id = Some("foobar");
+
+html! {
+    <div id?=maybe_id></div>
+}
+```
+If the attribute is set to a `None` value, it will behave as though the
+attribute wasn't set.
```

Comment:
`None` is already the value.

```suggestion
If the attribute is set to `None`, it will behave as though it wasn't set.
```

Reply:
Could you also add a small example here which compares the macro HTML to the generated HTML output? I feel like that's a good way to verify whether the reader has understood the concept.